### ISSUE TITLE
Not Marked as ThreadSafe

### ISSUE THREAD

**Issue by fsmherrm** [chkal/gitlab-code-quality-plugin#4]
Hello,

is it possible to make the plugin threadsafe?

```
[WARNING] *****************************************************************
[WARNING] * Your build is requesting parallel execution, but project      *
[WARNING] * contains the following plugin(s) that have goals not marked   *
[WARNING] * as @threadSafe to support parallel building.                  *
[WARNING] * While this /may/ work fine, please look for plugin updates    *
[WARNING] * and/or request plugins be made thread-safe.                   *
[WARNING] * If reporting an issue, report it against the plugin in        *
[WARNING] * question, not against maven-core                              *
[WARNING] *****************************************************************
[WARNING] The following plugins are not marked @threadSafe in gws-report-batch:
[WARNING] de.chkal.maven:gitlab-code-quality-plugin:1.0.1
[WARNING] Enable debug to see more precisely which goals are not marked @threadSafe.
[WARNING] *****************************************************************
```

Thank you and best regards
Marc

**Comment by chkal** [comment]
@fsmherrm Sure, I think this would be possible. Are you interested in providing a pull request?

**Comment by chkal** [comment]
Nevermind, I already figgured out how to fix this:

**Comment by chkal** [comment]
The issue is fixed in version 1.0.2, which just got released. It should show up in Maven Central in a few hours.
